### DISCUSSION TITLE
help me understand how integer file "BlockStates" are related to block positions

### DISCUSSION THREAD

**Discussion by blawald** [maruohon/litematica#720] [Q&A]
help me understand how integer file "BlockStates" are related to block positions. 
for example: here is my simple 4 block build
![2023-06-21_18 37 26](https://github.com/maruohon/litematica/assets/61388850/f7087612-078d-4d1d-aa4f-a01fad48b644)
How do these numbers relate to construction?
How do these numbers come from the coordinates of these blocks?
![image](https://github.com/maruohon/litematica/assets/61388850/f7bb2496-60c9-4b82-9958-c19c9c02989e)

**Comment by blawald** [comment]
please help me understand how integer file "BlockStates" are related to block positions**

**Comment by maruohon** [comment]
See #53.

Here are also screenshots of some of the conversations on my Discord where the format has been explained:
![image](https://github.com/maruohon/litematica/assets/2331050/cd3897c5-83b4-4e88-86a5-f248af090bf8)

![image](https://github.com/maruohon/litematica/assets/2331050/9578ed47-73cd-41bc-b488-609a7482cc75)

![image](https://github.com/maruohon/litematica/assets/2331050/67a92b2b-1c04-4247-8adf-62503a170bfd)

![image](https://github.com/maruohon/litematica/assets/2331050/2c891e42-a324-462b-aeb9-9d906c7b5643)

And here is another older conversation:
![image](https://github.com/maruohon/litematica/assets/2331050/6837cf69-f11f-472a-a18e-7fe7c74cf5cf)

![image](https://github.com/maruohon/litematica/assets/2331050/efcb702f-bf29-4427-8bc0-cb378b12f01d)

And another snippet:
![image](https://github.com/maruohon/litematica/assets/2331050/964713ba-06d4-4754-9652-fa6ad2d1c1cf)
